CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 0.6.0 - 2023-02-14
+* [maintenance] Support typeless endpoint for ES version 7.x & 8.x [#71](https://github.com/embulk/embulk-output-elasticsearch/pull/71)
```

Comment:
I think it should be 8x only because the version 7x can still run fine with the old endpoints

Reply:
I updated it. Please have a look again. Thank you.